src/main/java/org/lookout_studios/meals_management_system/meals_management_system/EstablishConnectionWithDatabase.java

```diff
@@ -0,0 +1,37 @@
+package org.lookout_studios.meals_management_system.meals_management_system;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+class accessData {
+    static String url = "jdbc:mysql://localhost:3306/mms_database";
+    static String username = "root";
+    static String password = "";
+    static String connectionInstanceName = "com.mysql.cj.jdbc.Driver";
```

Comment:
Avoid using magic values and **under no circumstances** include credentials in the source code. Create a separate config file for that and **do not** include it in your commits (include the file in `.gitignore`. Include instructions in the README file on where to enter the credentials.